cmd/consensus/main.go

```diff
@@ -104,8 +105,8 @@ func main() {
 			flags.Float64Var(&hotstuffTimeoutDecreaseFactor, "hotstuff-timeout-decrease-factor", timeout.DefaultConfig.TimeoutDecrease, "multiplicative decrease of timeout value in case of progress")
 			flags.Float64Var(&hotstuffTimeoutVoteAggregationFraction, "hotstuff-timeout-vote-aggregation-fraction", 0.6, "additional fraction of replica timeout that the primary will wait for votes")
 			flags.DurationVar(&blockRateDelay, "block-rate-delay", 500*time.Millisecond, "the delay to broadcast block proposal in order to control block production rate")
-			flags.BoolVar(&requireOneApproval, "require-one-approval", false, "require one approval per chunk when sealing execution results")
 			flags.UintVar(&chunkAlpha, "chunk-alpha", chmodule.DefaultChunkAssignmentAlpha, "number of verifiers that should be assigned to each chunk")
+			flags.UintVar(&requiredChunkApprovals, "required-chunk-approvals", validation.DefaultRequiredChunkApprovals, "number of approvals that are required for each chunk")
```

Comment:
The `DefaultRequiredChunkApprovals` is set to be `1` here. I'm a bit concerned to launch with this value, because we would be committing to a big step that introduces all the verification logic immediately. If things went wrong, we would have to ask all nodes to add flag with `require-chunk-approvals=0`, which is not easy to control

Instead, I would suggest to take smaller steps:
1. first launch with the default value as `0`
2. if step `1` works well, we turn on this flag for a few nodes: `require-chunk-approvals=1`
3. if step `2` went well, we roll it out to all our nodes, and potentially all partners.
4. if step `2` didn't go well, we could debug with only 1-2 nodes having this flag on, without affecting the sealing speed.

If everything went well for an entire spork, we will have confidence to set the default to be `1` for the next spork.

Thoughts?

Reply:
We synced today on the strategy of rolling this into production:
* please see notion doc https://www.notion.so/dapperlabs/Sealing-Verification-Grooming-1aaa12b552b9497f9c62e76ec2d948db#986201016c47456da805318c9b7eecf8
* there is an an issue for the corresponding work: https://github.com/dapperlabs/flow-go/issues/5180

Would suggest to just leave this aspect as is in this PR and include all the necessary changed for https://github.com/dapperlabs/flow-go/issues/5180 in a new PR.